README.md

```diff
@@ -1,17 +1,38 @@
-# Sample
+﻿# ABOUT
```

Comment:
We don't want to be shouting at the user. Change this to `About`. This applies to every other heading as well.